### DISCUSSION TITLE
code execution issue after validaton success

### DISCUSSION THREAD

**Discussion by deep80** [livewire/livewire#8802] [1. Help]
I got to perform a form submission. for that i have created a separate form class as per the documentation. there i have added the properties needed for that form with validation.

The validation is working fine. however it fails to execute the code after successful validation.

Here's the form submission method in my component class:

public function save()
{
        $this->form->validate();
        dd('hello world');
}

after validation i am not able to see the dd() execution.

**Comment by Fulue** [comment]
There is no `$this->form->validate()` method in Livewire, so you need to use the standard `validate()` method in the Livewire component to perform data validation.

**Comment by deep80** [comment]
This is primarily due to the reason that I imported the form class into my
component and called the $this->form->validate() method. I have imported
the form class as $form. Anyways I have figured out how to solve that and
finally achieved what I wanted
